error: user substs: UserSubsts { substs: [Ref(ReStatic, Uint(U32), Not)], user_self_ty: None }
  --> $DIR/dump-fn-method.rs:28:13
   |
LL |     let x = foo::<&'static u32>;
   |             ^^^^^^^^^^^^^^^^^^^

error: user substs: UserSubsts { substs: [Bound(DebruijnIndex(0), BoundTy { var: 0, kind: Anon }), Uint(U32), Bound(DebruijnIndex(0), BoundTy { var: 1, kind: Anon })], user_self_ty: None }
  --> $DIR/dump-fn-method.rs:34:13
   |
LL |     let x = <_ as Bazoom<u32>>::method::<_>;
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: user substs: UserSubsts { substs: [Uint(U8), Ref(ReStatic, Uint(U16), Not), Uint(U32)], user_self_ty: None }
  --> $DIR/dump-fn-method.rs:43:13
   |
LL |     let x = <u8 as Bazoom<&'static u16>>::method::<u32>;
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: user substs: UserSubsts { substs: [Bound(DebruijnIndex(0), BoundTy { var: 0, kind: Anon }), Bound(DebruijnIndex(0), BoundTy { var: 1, kind: Anon }), Uint(U32)], user_self_ty: None }
  --> $DIR/dump-fn-method.rs:51:5
   |
LL |     y.method::<u32>(44, 66);
   |     ^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 4 previous errors

